report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,84 @@
+% point of document
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is.
+% - What "el-teknik" is.
+% - Document out findings of the student and teacher interviews.
+
+\section{Arduino and their usage in danish education}
+During the third year of the HTX\footnote{Higher Technical Examination Programme, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practice. 
```

Comment:
"Programme" is technically correct, however it is the British way of spelling it, and everywhere else we use the American way.
Change it to "Program"